format.go

```diff
@@ -16,7 +13,7 @@ const (
 type Units uint
 
 const (
-	UnitNone Units = iota
```

Comment:
I have not had the chance of using mpb yet. I discovered it yesterday looking at pb and I liked it.